GameLogic/Runtime/GamePreparation.cs

```diff
@@ -14,10 +15,10 @@ public class PreparationLogic
         public void initialize() {
```

Comment:
Mb it would be possible to move into the class constructor instead of have in separate method?

Reply:
fixed